core-api/src/main/java/com/optimizely/ab/config/audience/UserAttribute.java

```diff
@@ -29,9 +29,9 @@
 
     private final String name;
     private final String type;
-    private final String value;
+    private final Object value;
 
-    public UserAttribute(@Nonnull String name, @Nonnull String type, @Nullable String value) {
+    public UserAttribute(@Nonnull String name, @Nonnull String type, @Nullable Object value) {
```

Comment:
@thomaszurkan-optimizely do we need to overload this constructor to allow you to pass in a `String value`? I don't think this change will break current usage of the class.

Reply:
Nevermind, I think this will be fine as is. Can ignore my previous comment.